src/panels/lovelace/strategies/original-states-strategy.ts

```diff
@@ -86,7 +86,7 @@ export class OriginalStatesStrategy {
     info: Parameters<LovelaceDashboardStrategy["generateDashboard"]>[0]
   ): ReturnType<LovelaceDashboardStrategy["generateDashboard"]> {
     return {
-      title: info.hass.config.location_name,
```

Comment:
When is this not the case?

Reply:
Not sure exactly. Possibly when not in storage mode?

Reply:
No. `config` is part of the `hass` object and we do not show the UI unless it's loaded: https://github.com/home-assistant/frontend/blob/dev/src/layouts/home-assistant.ts#L69

Reply:
Not sure then how this error is even showing. I will leave this open and try to trace it back to find the root issue